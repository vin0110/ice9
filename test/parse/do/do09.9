do a -> do b -> do c -> d(); od od od
