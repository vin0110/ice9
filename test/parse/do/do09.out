DO:
  ID:'a'
(body)
  DO:
    ID:'b'
  (body)
    DO:
      ID:'c'
    (body)
      EXP:
        EXP:
          CALL:
            ID:'d'
          (arglist)
            nil
